doc/release-notes/4.16-release-notes.md

```diff
@@ -0,0 +1,86 @@
+
+### HTML Codebook Export
+
+Researchers will now be able to download HTML Codebooks as an additional Dataset Export type. This codebook provides valuable information about the contents and structure of a dataset and will increase reusability of the datasets in Dataverse.
+
+### Harvesting Improvements
+
+The Harvesting code will now better handle problematic records during incremental harvests. Fixing this will mean not only fewer manual interventions by installation administrators to keep harvesting running, but it will also mean researchers can more easily find and access data that is important to their research.
+
+## Major Use Cases
+
+Newly-supported use cases in this release include:
+
+- As a researcher, I can view the works that have cited a dataset.
+- As a researcher, I can view the downloads and views for a dataset, based on the Make Data Count standard.
+- As a researcher, I can export an HTML codebook for a dataset.
+- As a researcher, I can expect harvested datasets to be made available more regularly.
+- As a researcher, I'll encounter fewer locks as I go through the publishing process.
+- As an installation administrator, I no longer need to destroy a PID in another system after destroying a dataset in Dataverse.
+
+## Notes for Dataverse Installation Administrators
+
+### Run ReExportall
+
+We made changes to the citation block in this release that will require installations to run ReExportall as part of the upgrade process. We've included this in the detailed instructions below.
+
+### Custom Analytics Code Changes
+
+You should update your custom analytics code to include CDATA sections, inside the `script` tags, around the javascript code. We have updated the documentation and sample analytics code snippet provided in [Installation Guide > Configuration > Web Analytics Code](http://guides.dataverse.org/en/latest/installation/config.html#web-analytics-code) to fix a bug that broke the rendering of the 403 and 500 custom error pgs (#5967).
+
+### Destroy Updates
+
+Destroying Datasets in Dataverse will now unregister/delete the PID with the PID provider. This eliminates the need for an extra step to "clean up" a PID registration after destroying a Dataset.
+
+### Deleting Notifications
+
+In making the fix for #5687 we discovered that notifications created prior to 2018 may have been invalidated. With this release we advise that these older notifications are deleted from the database. The following query can be used for this purpose:
+
+`delete from usernotification where date_part('year', senddate) < 2018;`
+
+### Lock Improvements
+
+In 4.15 a new lock was added to prevent parallel edits. After seeing that the lock was not being released as expected, which required administrator intervention, we've adjusted this code to release the lock as expected.
+
+For the complete list of code changes in this release, see the <a href="https://github.com/IQSS/dataverse/milestone/83?closed=1">4.16 milestone</a> in Github.
```

Comment:
Above this "for the complete list" line, I would consider adding a header called "### Complete List of Chances" or something. Otherwise it feels like I'm still reading about lock improvements.

Reply:
Looks good! I agree with @pdurbin’s formatting comment. All is clear, except I’d like a little more explanation of HTML code books export. How does it provide value? As a researcher, what does exporting an HTML code book do for me? Also, how does the system create this? Does it require anything of the depositor to enable this functionality?

Reply:
@TaniaSchlatter good catch. There is very little written in the guides about HTML codebook export (see f6edfdd) and why it's valuable. Whatever we add to the release notes should also be added to the User Guide.

I can answer some of your more technical questions.

The system creates the HTML codebook on the fly. You can play with it at http://phoenix.dataverse.org

No one has to do anything to enable it. It "just works" like the other export formats.

Reply:
Thanks both for the feedback. I implemented the new header in https://github.com/IQSS/dataverse/commit/34e69c507ae4889b6ed3e01a9c07a6c9931e01a7.

@jggautier is reworking the codebook section of the docs and we'll add some of that content in this space this afternoon.

Reply:
Go, @jggautier go!